### DISCUSSION TITLE
Welcome to POGOProtos Discussions!

### DISCUSSION THREAD

**Discussion by Furtif** [Furtif/POGOProtos#40] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shiny-Grid** [comment]
Hello I am new to coding only had 2 electives during college and non of them was python. That being said I am very curious about your project with pokemon go. I love playing this game and would love to know where can i go to gain the insight to use your code to tinker with the game if that is possible? From what i can tell it seems like this is a bridge/library to interact with the game maybe. Not sure where to start.

**Comment by Furtif** [comment]
let's say no one cares
